as/byte-buffer.ts

```diff
@@ -0,0 +1,365 @@
+import { FILE_IDENTIFIER_LENGTH, SIZEOF_INT } from './constants';
+import { Long } from './long';
```

Comment:
is it just for compatibility with JS API? Otherwise AssemblyScript support `i64/u64` and don't require this polyfill

Reply:
Yeah, it was just for JS compatibility.